edge-inn-reach/src/main/resources/volaris/edge-inn-reach/features/inn-reach-proxy.feature

```diff
@@ -0,0 +1,22 @@
+@ignore
+@parallel=false
+Feature: Get authentication token
+
+  Background:
+    * url baseUrl
+
+    * print 'Create central servers'
+    * callonce read(featuresPath + 'central-server.feature@create') {'okapitokenUser': #(okapitokenUser)}
+
+    * print 'Create JWT Token : Get Token'
+    * callonce read(globalPath + 'jwt-token-helper.feature@GetJWTToken')
+    * def responseToken = 'Bearer ' + response.access_token
+    * def headersUser = { 'Content-Type': 'application/json', 'Authorization' : '#(responseToken)', 'x-to-code': 'fli01', 'x-from-code': '69a3d', 'Accept': 'application/json'  }
+
+  Scenario: Negative scenario for JWT token
+    * print 'Negative scenario for JWT token'
+    * callonce read(globalPath + 'jwt-token-helper.feature@GetInvalidJWTToken')
+
+  Scenario: Delete central server
```

Comment:
Delete central server should not be present in proxy file , It should be in transaction file , please check FAT-1569

Reply:
removed